.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.14.0
+current_version = 2.17.0
```

Comment:
I'm guess we haven't been merging back recently.

Reply:
We have, but I don't think this file was bumped; only `setup.py`